data/data-pipeline/data_pipeline/tests/score/fixtures.py

```diff
@@ -2,12 +2,218 @@
 import pytest
 from data_pipeline.config import settings
 from data_pipeline.score import field_names
+from data_pipeline.etl.score import constants
+
+GEOID_TRACT_FIELD_NAME = field_names.GEOID_TRACT_FIELD
 
 
 @pytest.fixture(scope="session")
 def final_score_df():
     return pd.read_csv(
         settings.APP_ROOT / "data" / "score" / "csv" / "full" / "usa.csv",
-        dtype={field_names.GEOID_TRACT_FIELD: str},
+        dtype={GEOID_TRACT_FIELD_NAME: str},
+        low_memory=False,
+    )
+
+
+@pytest.fixture()
+def census_df():
+    census_csv = constants.DATA_PATH / "dataset" / "census_acs_2019" / "usa.csv"
+    return pd.read_csv(
+        census_csv,
+        dtype={GEOID_TRACT_FIELD_NAME: "string"},
+        low_memory=False,
+    )
+
+
+@pytest.fixture()
+def ejscreen_df():
+    ejscreen_csv = constants.DATA_PATH / "dataset" / "ejscreen" / "usa.csv"
+    return pd.read_csv(
+        ejscreen_csv,
+        dtype={GEOID_TRACT_FIELD_NAME: "string"},
+        low_memory=False,
+    )
+
+
+@pytest.fixture()
+def hud_housing_df():
+    hud_housing_csv = (
+        constants.DATA_PATH / "dataset" / "hud_housing" / "usa.csv"
+    )
```

Comment:
Idea: why re-write the paths for each dataset, when there's a class variable for it [here](https://github.com/usds/justice40-tool/blob/main/data/data-pipeline/data_pipeline/etl/sources/hud_housing/etl.py#L10)? I'm mostly thinking about keeping it DRY?

Reply:
(this would apply to all the df's here)

Reply:
They're all instance variables and it just didn't make sense to me to import and instantiate all those classes to just get the paths. I tried to factor them up as classvars but they rely on a method so like it didn't make a ton of sense. So in this case I figured keeping the coupling lower and relying on a pretty thorough convention was worth the repetition. Open to arguments otherwise, but that as my logic.

Reply:
Makes sense, Matt, thanks! I've seen this repetition pattern in fixtures often to totally decouple things.

Reply:
:bowing_man: